### PR TITLE
Replace CustomFormats tutorial with a more detailed one

### PR DESCRIPTION
**Pull request by rimadoma**
I had to study quite a bit how to create a new SCIFIO  file format even after reading the tutorials, so decided to write a new one to make it easier for others.

### CODE REVIEW

**Comment by ctrueden** [comment]
Awesome, Richard. I will try to look this over and comment and/or merge it later this week.

**Comment by rimadoma** [comment]
Planes are read incorrectly in the tutorial, not ready to merge

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,14 @@
 .DS_Store
-.classpath
-.project
-.settings
-target
 test-output
+
+# Eclipse #
+/.classpath/
```
Comment (ctrueden) [inline .gitignore]: The file `.classpath` is not a directory, and thus should not have a trailing slash.

File: .gitignore [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+/.project/
```
Comment (ctrueden) [inline .gitignore]: The file `.project` is not a directory, and thus should not have a trailing slash.

File: .gitignore [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
+/.settings/
+
+# Maven #
+*/target/
+
+# IntelliJ #
+/.idea/
+*.iml
```
Comment (ctrueden) [inline .gitignore]: These files only appear in the root directory of the repository, right? So it should be `/*.iml`.

**Comment by ctrueden** [comment]
What I want to do is to take your new tutorial and make it a new repository called `example-scifio-format`. This would be similar to the `imagej/example-imagej-command` and `imagej/example-legacy-plugin` repositories, in that they are intended to be starting point templates for your own work.

I have now created this repository, and invited you as a collaborator. Would you please recast this tutorial in that light, pushing it to the master branch? I think it will help people a lot to have it decoupled as its own project, from which they can make their own formats.

The other reason we are doing this is because for API _usage_ tutorials, we are moving to scientific notebooks, as you probably saw in `imagej/tutorials`. We will be migrating all the SCIFIO API usage stuff into an "Introduction to SCIFIO" notebook in `imagej/tutorials` later this year. This `scifio-tutorials` repository will then be deleted, once it is an empty shell.

File: .gitignore (hunk 3 above)
Comment (rimadoma) [inline .gitignore]: If you have a multi-module project, each sub-module has an *.iml file at it's root

Comment (ctrueden) [inline .gitignore]: Oh, yeah, this _is_ a multi-module project, isn't it? I was thinking "we don't have any of _those_!"

**Pull request closed without merging**